app/models/educator.rb

```diff
@@ -70,11 +70,6 @@ def labels
     educator_labels.map(&:label_key)
   end
 
-  def default_homeroom
```

Comment:
This method raised an error in the test suite. When I searched for implementations of `:default_homeroom` in our app using Sublime, nothing turned up. Removing it didn't affect any other areas of the test suite. So I think this is unused and safe to remove.